feat(ui): created item display page

mauriprado opened the pull request.

mauriprado committed:
feat(ui): created item display

dalbitresb12 commented on android/build.gradle:
```diff
@@ -26,6 +26,6 @@ subprojects {
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```
Is this change needed?

dalbitresb12 commented on lib/views/item_view.dart:
```diff
@@ -0,0 +1,163 @@
+                TextSpan(
+                  children: <TextSpan>[
+                    TextSpan(text: "Men's t-shirt        ", style: TextStyle(
+                        fontSize: 30.0,
```
You should use some margin here instead of spaces.

dalbitresb12 commented on lib/main.dart:
```diff
@@ -21,7 +22,7 @@ class FitsterApp extends StatelessWidget {
         primarySwatch: Colors.blue,
       ),
-      home: const ObjectDetectorView(title: title),
+      home: ItemView(),
```
After you finish changing everything, return this to the `ObjectDetectorView`.

dalbitresb12 reviewed (changes requested):
Please fix the warnings from the Dart analysis workflow and run the auto-formatter. Make sure to add trailing commas to get nice formatting.

Most of the warnings come from using double quotes (`"`) instead of single quotes (`'`) and for not using `const` constructors, but double check in the workflow results for more info or in your IDE.

dalbitresb12 commented:
The PR title is good btw, it's just that the check of it is broken. Pushing new commits will run the fixed version of the PR title checker which should accept the title.

dalbitresb12 committed:
revert: `android/build.gradle` to 5d4c96d353576c3f527fdebd2e315fdf4b702b5a

dalbitresb12 committed:
revert: `lib/main.dart` to 3ee1625542e8f276636b83f16d14ece53d04fb11

dalbitresb12 committed:
revert: package manifest files

- `pubspec.yaml` to c8ea71d8f28aaf70164b62ba56ecda3cec84d50a
- `pubspec.lock` to 5d4c96d353576c3f527fdebd2e315fdf4b702b5a

dalbitresb12 committed:
build: added flutter_rating_bar package

dalbitresb12 committed:
style: moved button to custom widget and formatting

dalbitresb12 reviewed (dismissed):
LGTM, did requested changes myself.

dalbitresb12 committed:
Merge branch 'main' into feat/item-screen

dalbitresb12 committed:
build: update flutter to v3.10.5

Dependencies got auto-updated with the Flutter update

dalbitresb12 committed:
fix: use `const` construction for `Row`

`const` constructors for the `Row` widget were just added in the latest Flutter update

dalbitresb12 reviewed (approved):
LGTM
